clang/test/CodeGenSYCL/kernel-caller-entry-point.cpp

```diff
@@ -91,12 +89,6 @@ int main() {
 // CHECK-HOST-WINDOWS-NEXT: }
 
 // Verify that SYCL kernel caller functions are emitted for each device target.
-//
-// FIXME: The following set of matches are used to skip over the declaration of
```

Comment:
I think this may cause conflicts that will be harder to figure out when upstream starts cutting device code too. Perhaps disabling checks with a comment to the pulldown coordinators will be slightly easier to understand.

Reply:
Thanks @Fznamznon. I'm going to see if I can land a change upstream to remove the need for these lines. That should help to minimize any impact to pulldown coordinators.